src/mscorlib/model.xml

```diff
@@ -2326,6 +2355,17 @@
       <Member MemberType="Property" Name="InvalidCultureName" />
       <Member MemberType="Property" Name="Message" />
     </Type>
+    <Type Name="System.Globalization.CultureTypes">
+      <Member MemberType="Field" Name="AllCultures" />
```

Comment:
@tarekgh I'm happy to chat more with you offline if you like but the general approach is to expose all the APIs unless there is a strong reason we cannot. Even in some edge case APIs that don't work on some platforms we will still expose and have them throw.